mrp_multi_level/wizards/mrp_multi_level.py

```diff
@@ -507,7 +508,7 @@ def _mrp_initialisation(self, mrp_areas):
         init_counter = 0
         for mrp_area in mrp_areas:
             for product_mrp_area in product_mrp_areas.filtered(
-                lambda a: a.mrp_area_id == mrp_area
+                lambda a, mrp_area=mrp_area: a.mrp_area_id == mrp_area
```

Comment:
why?

Reply:
it was failing some pre-commit error, that's why need to declare a variable used within lambda function

Reply:
Ugh, pre-commit each time is adding more features... haha